yarn.lock

```diff
@@ -3995,7 +3995,7 @@
   dependencies:
     "@types/node" "*"
 
-"@types/lodash@^4.14.161", "@types/lodash@^4.17.0":
+"@types/lodash@^4.17.0":
```

Comment:
I don't know who you are but I will look for you, I will find you and I will kill you.

Reply:
😂 It's in the docs I think

Reply:
Also async-storage